Gemfile

```diff
@@ -110,7 +110,7 @@ gem 'rack', '~> 1.6.0'
 gem 'rack-ssl', '~> 1.4.0'
 gem 'rack-utf8_sanitizer', '~> 1.3.0'
 gem 'rails-i18n', '~> 4.0.0', '< 5.0.0'
-gem 'recaptcha', '~> 0.4.0', '< 0.5.0', :require => 'recaptcha/rails'
+gem 'recaptcha', '~> 4.9.0', :require => 'recaptcha/rails'
```

Comment:
Should be `gem 'recaptcha', '~> 4.9.0', '< 4.10.0', :require => 'recaptcha/rails'` to signify that we _can't_ go to `4.10.0'` or greater.